server/game/cards/15-DotE/Drogon.js

```diff
@@ -11,13 +12,11 @@ class Drogon extends DrawCard {
             when: {
                 afterChallenge: event => event.challenge.winner === this.controller && this.isAttacking()
             },
-            handler: context => {
-                let characters = this.game.currentChallenge.defendingPlayer.filterCardsInPlay(card => card.getType() === 'character' && card.getStrength() <= 1);
-                this.game.killCharacters(characters);
-
-                this.game.addMessage('{0} uses {1} to kill all characters {2} controls with STR 1 or lower',

```

Comment:
Need to convert this to use `message`. Currently, no message will be printed anymore.